src/main/java/org/sefglobal/scholarx/controller/AuthUserController.java

```diff
@@ -47,10 +47,10 @@ public Object getLoggedUser(Authentication authentication)
 
   @GetMapping("/programs/mentor")
   @ResponseStatus(HttpStatus.OK)
-  public List<Program> getMentoringPrograms(Authentication authentication)
+  public List<Program> getMentoringPrograms(Authentication authentication, @RequestParam(required = false) EnrolmentState state)
```

Comment:
```suggestion
  public List<Program> getMentoringPrograms(Authentication authentication, @RequestParam(required = false) EnrolmentState mentorState)
```
Make the var meaningful

Reply:
Sure